game/src/main/org/apollo/game/sync/block/SynchronizationBlock.java

```diff
@@ -44,11 +44,11 @@ public static SynchronizationBlock createAppearanceBlock(Player player) {
 	 * Creates a {@link ChatBlock} for the specified player.
 	 *
 	 * @param player The player.
-	 * @param chatMessage The chat message.
+	 * @param publicChatMessage The chat message.
 	 * @return The chat block.
 	 */
-	public static SynchronizationBlock createChatBlock(Player player, ChatMessage chatMessage) {
-		return new ChatBlock(player.getPrivilegeLevel(), chatMessage);
+	public static SynchronizationBlock createChatBlock(Player player, PublicChatMessage publicChatMessage) {
```

Comment:
`publicChatMessage` -> `message` please (I prefer one word variable names unless the longer form is really necessary)